src/ansys/dpf/post/zone.py

```diff
@@ -0,0 +1,68 @@
+"""Module containing the ``Zone`` class and sub-classes.
+
+Zone
+----
+
+"""
+from abc import ABC
+from typing import List
+
+
+class Zone(ABC):
+    """Base class for fluid Zone objects."""
+
+    def __init__(self, name: str, id: int, scoping):
+        """Initialize this class."""
+        self._name = name
+        self._id = id
+        self._mesh_scoping = scoping
+
+    @property
+    def name(self) -> str:
+        """Return the name of the Zone."""
+        return self._name
+
+    @property
+    def id(self) -> int:
+        """Return the ID of the Zone."""
+        return self._id
+
+
+class CellZone(Zone):
+    """Fluid zone defined on cells."""
+
+    @property
+    def cells(self) -> List[int]:
+        """Return the list of IDs of the cells in the Zone."""
+        return self._mesh_scoping.ids
+
+
+class FaceZone(Zone):
+    """Fluid zone defined on faces."""
+
+    @property
+    def faces(self) -> List[int]:
+        """Return the list of IDs of the faces in the Zone."""
+        return self._mesh_scoping.ids
+
+
+class Zones:
+    """List of fluid zones."""
+
+    def __init__(self):
```

Comment:
do we have a way to access zones by name?

Reply:
@cbellot000 once the MeshInfo is available, sure, but not now